app/lib/one_sky.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'rest-client'
+require 'auth_util'
+require 'hash_util'
+
+module OneSky
+  # @return [Hash] of translated phrases from OneSky
+  def self.download_translated_phrases(filename, project_id:, language_code:)
+    logger.info "Downloading translated phrases for: #{filename} with language: #{language_code}"
+
+    response = RestClient.get "https://platform.api.onesky.io/1/projects/#{project_id}/translations",
+                              params: headers(language_code).merge(
+                                source_file_name: filename, export_file_name: filename
+                              )
+
+    # NOTE: maybe Error::TextNotFoundError should have been a error under a OneSky namespace
+    raise Error::TextNotFoundError, 'No translated phrases found for this language.' if response.code == 204
```

Comment:
@kares I know this wasn't originally part of the issue/PR and was the original behavior, but I think if OneSky returns no translations we should return an empty hash instead of throwing an error here.

If a phrase that is required is missing, it will throw an error here: https://github.com/CruGlobal/mobile-content-api/pull/177/files#diff-7077e27c99fca731384a65b65766dce2R20

I'll leave it up to you if you want to include that change in this PR or as a small PR after this one is merged

Reply:
the reasoning behind relaxing this is that if a page is published that doesn't have any OneSky phrases to translate, we currently need to load a placeholder phrase so that we can publish the page. We shouldn't be dependent on having translations in OneSky if we don't actually use them.